var/spack/repos/builtin/packages/votca/package.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Votca(CMakePackage):
+    """VOTCA is a software package which focuses on the analysis of molecular
+       dynamics data, the development of systematic coarse-graining techniques
+       as well as methods used for simulating microscopic charge (and exciton)
+       transport in disordered semiconductors.
+    """
+    homepage = "https://www.votca.org"
+    url      = "https://github.com/votca/votca/tarball/v2022-rc.1"
+    git      = "https://github.com/votca/xtp.git"
+    maintainers = ['junghans']
+
+    version('master', branch='master')
+    version('stable', branch='stable')
+    version('2022-rc.2', sha256='eefde51470ec1437d0127fb02c2745f33e434deff53cdaee97691c36ce447fb1')
+    version('2022-rc.1', sha256='d53ca9fde364a97d91bf3bed15223536ffa598b2dec7bccd459accae265391b1')
+
+    variant('mkl', default=False, description='Build with MKL support')
+    variant('new-gmx', default=False, description='Build against gromacs>2019 - no tabulated kernels')
```

Comment:
I wonder how useful this variant actually is. I think it's relatively easy to express this in Spack's spec syntax. For example, the following:
```console
$ spack install votca+new-gmx
$ spack install votca~new-gmx
```
is equivalent to:
```console
$ spack install votca^gromacs@2020:
$ spack install votca^gromacs@:2019
```

Reply:
I know my users, the latter might be a bit too complicated for most. I want it to work by default and means gromacs-2019 is required. If gromacs ever re-implements tabulated interactions we can drop it again. (see https://gitlab.com/gromacs/gromacs/-/issues/1347)